profiles/VP_ANDROID_15.json

```diff
@@ -0,0 +1,205 @@
+                    }
+                }
+            },
+            "formats": {
+                "VK_FORMAT_A4B4G4R4_UNORM_PACK16_EXT": {
+                    "VkFormatProperties": {
+                        "linearTilingFeatures": ["VK_FORMAT_FEATURE_COLOR_ATTACHMENT_BIT", "VK_FORMAT_FEATURE_BLIT_DST_BIT", "VK_FORMAT_FEATURE_TRANSFER_SRC_BIT", "VK_FORMAT_FEATURE_TRANSFER_DST_BIT"],
+                        "optimalTilingFeatures": ["VK_FORMAT_FEATURE_SAMPLED_IMAGE_BIT", "VK_FORMAT_FEATURE_COLOR_ATTACHMENT_BIT", "VK_FORMAT_FEATURE_BLIT_SRC_BIT", "VK_FORMAT_FEATURE_BLIT_DST_BIT", "VK_FORMAT_FEATURE_TRANSFER_SRC_BIT", "VK_FORMAT_FEATURE_TRANSFER_DST_BIT"],
+                        "bufferFeatures": []
+                    }
+                },
+                "VK_FORMAT_A4R4G4B4_UNORM_PACK16_EXT": {
+                    "VkFormatProperties": {
+                        "linearTilingFeatures": ["VK_FORMAT_FEATURE_COLOR_ATTACHMENT_BIT", "VK_FORMAT_FEATURE_BLIT_DST_BIT", "VK_FORMAT_FEATURE_TRANSFER_SRC_BIT", "VK_FORMAT_FEATURE_TRANSFER_DST_BIT"],
+                        "optimalTilingFeatures": ["VK_FORMAT_FEATURE_SAMPLED_IMAGE_BIT", "VK_FORMAT_FEATURE_COLOR_ATTACHMENT_BIT", "VK_FORMAT_FEATURE_BLIT_SRC_BIT", "VK_FORMAT_FEATURE_BLIT_DST_BIT", "VK_FORMAT_FEATURE_TRANSFER_SRC_BIT", "VK_FORMAT_FEATURE_TRANSFER_DST_BIT"],
+                        "bufferFeatures": []
+                    }
+                }
+            }
+        },
+        "DepthClipControl": {
+            "extensions": {
+                "VK_EXT_depth_clip_control": 1
+            },
+        },
+        "maxGeometryTotalOutputComponents": {
+            "features": {
+                "properties": {
+                    "VkPhysicalDeviceProperties": {
+                        "limits": {
+                            "maxGeometryTotalOutputComponents": 1028,
+                        }
+                    }
+                }
+            },
+        },
+        "primitivesGeneratedQuery": {
+            "extensions": {
+                "VK_EXT_primitives_generated_query": 1
+            },
+            "features": {
+                "VkPhysicalDevicePrimitivesGeneratedQueryFeaturesEXT": {
+                    "primitivesGeneratedQuery": true
+                }
+            }
+        },
+        "pipelineStatisticsQuery": {
+            "features": {
+                "VkPhysicalDeviceFeatures": {
+                    "pipelineStatisticsQuery": true
+                }
+            }
+        },
+        "androidYUVtarget": {
+            "extensions": {
+                "VK_ANDROID_external_format_resolve": 1
+            }
+        },
+        "generalYUVtarget": {
+            "extensions": {
+                "VK_KHR_ycbcr_rendertarget": 1
```

Comment:
This extension doesn't exist even in Vulkan Header 1.3.268, I didn't manage to find an alternative.

Reply:
A spec MR for this extension was created, but it has not yet progressed.  Samsung (who created this extension) asked that we have an either-or, so that when they (and other Khronos members) finish this extension, it will be a valid/proper superset of and substitute of the `VK_ANDROID_external_format_resolve` extension.

Reply:
The profile solution, including the JSON schema is generated from `vk.xml` so the MR need to be merge. Note this is a public repository, not sure whether this conversation is ok here.